features/salt_formulas.feature

```diff
@@ -57,6 +69,14 @@ Feature: Use salt formulas
      And I click on "Clear values" and confirm
      And I click on "Save Formula"
      Then I should see a "Formula saved!" text
+     And I refresh the pillar data
+     And the pillar data for "formulas" should be "- locale" on "sle-minion"
+     And the pillar data for "timezone:name" should be "CET" on "sle-minion"
+     And the pillar data for "keyboard_and_language:keyboard_layout" should be "English (US)" on "sle-minion"
+     And the pillar data for "keyboard_and_language:language" should be "English (US)" on "sle-minion"
+     And the pillar data for "formulas" should be empty on "ceos-minion"
+     And the pillar data for "timezone" should be empty on "ceos-minion"
```

Comment:
@meaksh ,

looking at your test here:

https://github.com/SUSE/spacewalk-testsuite-base/pull/295/files#diff-8aef6aa74851e0d058f469ae0e69142fR130

you run commands when the ceos-minion is not bootstrapped.

https://github.com/SUSE/spacewalk-testsuite-base/blob/master/run_sets/testsuite.yml#L71

Imho, we need a new feature only for that, i would put it at the end.